changelogs/changelog.yaml

```diff
@@ -1971,3 +1971,59 @@ releases:
           created using the same name as an existing uplink, then the existing
           uplink is modified.
     release_date: '2024-07-31'
+
+  9.6.0:
+    changes:
+      release_summary: '- The ``ome_application_certificate`` module is
```

Comment:
The ``ome_application_certificate`` module is enhanced to support the upload operation of one or more certificates on OpenManage Enterprise.

Reply:
Keep the current summary.